wbooks/src/App/redux/Auth/reducer.js

```diff
@@ -0,0 +1,28 @@
+import { AUTH_OK, AUTH_ERROR, LOGOUT } from './actions';
+
+const initialState = { isLoggedIn: false, errorMsg: '' };
+
+const auth = (state = initialState, action) => {
+  switch (action.type) {
+    case AUTH_OK:
+      return {
+        ...state,
+        isLoggedIn: true,
+        errorMsg: ''
+      };
+    case AUTH_ERROR:
```

Comment:
We name these `_SUCCESS` and `_FAILURE`.
Just for your information, no need to change them here.